.travis.yml

```diff
@@ -1,12 +1,13 @@
 language: python
 python:
-- '2.6'
 - '2.7'
-- '3.2'
```

Comment:
the point is to mention it explicitly for the debugging/reproducability
purpose, just to be sure what travis should use

On 16 March 2015 at 17:10, Patrick <notifications@github.com> wrote:

> In .travis.yml
> <https://github.com/pnpnpn/timeout-decorator/pull/16#discussion_r26499600>
> :
>
> >  - '2.7'
> > -- '3.2'
>
> Got it. In that case, should we remove the version from this yaml
> altogether and let travis use the default one?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pnpnpn/timeout-decorator/pull/16/files#r26499600>.
>



-- 
Anatoly Bubenkov